src/methods/nickname.js

```diff
@@ -21,7 +21,9 @@ const nickname = customer => {
  * @returns {string}
  *
  * @example
- * // TODO
+ * const custumer = { 'name': { 'given_name': 'Jonh', 'family_name': 'Lock' }, 'display_name': 'Lock' }
```

Comment:
```suggestion
 * const customer = { 'name': { 'given_name': 'Jonh', 'family_name': 'Lock' }, 'display_name': 'Locky' }
```

Reply:
I'm fixing the var name and changing the example object to make clearer that the method is using `display_name` first. Was ambiguous as `family_name` was equal to `display_name`...